### PR TITLE
Automatically port System.Text.Json.JsonProperty

### PR DESCRIPTION
**Pull request by carlossanlop**

### CODE REVIEW

File: xml/System.Text.Json/JsonProperty.xml [hunk 1]
```diff
@@ -211,9 +211,13 @@ This method is functionally equal to doing an ordinal comparison of `text` and <
         <Parameter Name="writer" Type="System.Text.Json.Utf8JsonWriter" />
       </Parameters>
       <Docs>
-        <param name="writer">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="writer">The writer into which the property is to be written.</param>
```
Comment (carlossanlop) [inline xml/System.Text.Json/JsonProperty.xml]: @rpetrusha modified the `writer` param in my other PRs to look like below, so I'll apply it:

```suggestion
        <param name="writer">The writer to which to write the property.</param>
```

File: xml/System.Text.Json/JsonProperty.xml [hunk 2]
```diff
@@ -211,9 +211,13 @@ This method is functionally equal to doing an ordinal comparison of `text` and <
         <Parameter Name="writer" Type="System.Text.Json.Utf8JsonWriter" />
       </Parameters>
       <Docs>
-        <param name="writer">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="writer">The writer to which to write the property.</param>
+        <summary>Writes this property into the specified writer as a named JSON object property.</summary>
```
Comment (carlossanlop) [inline xml/System.Text.Json/JsonProperty.xml]: Keeping this consistent with the recent @rpetrusha modifications to similar PRs:

```suggestion
        <summary>Writes the property to the provided writer as a named JSON object property.</summary>
```

Comment (mairaw) [reply xml/System.Text.Json/JsonProperty.xml]: Sounds good!

File: xml/System.Text.Json/JsonProperty.xml [hunk 3]
```diff
@@ -211,9 +211,13 @@ This method is functionally equal to doing an ordinal comparison of `text` and <
         <Parameter Name="writer" Type="System.Text.Json.Utf8JsonWriter" />
       </Parameters>
       <Docs>
-        <param name="writer">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="writer">The writer to which to write the property.</param>
```
Comment (ahsonkhan) [inline xml/System.Text.Json/JsonProperty.xml]: This is difficult to read. Maybe we could rephrase it, or not use `writer`.
```suggestion
        <param name="writer">The output to which to write the property.</param>
```

Or maybe like this:
```suggestion
        <param name="writer">The output to write the property into.</param>
```

**Pull request merged**